app/experimenter/static/rapid/components/App.tsx

```diff
@@ -1,15 +1,29 @@
 import React from "react";
-import { BrowserRouter, Switch, Route } from "react-router-dom";
+import { Switch, Link, Route } from "react-router-dom";
+
+import ExperimentFormPage from "experimenter-rapid/components/pages/ExperimentFormPage";
 
 const App: React.FC = () => {
+  React.useEffect(() => {
+    const subHeader = document.getElementById("sub-header");
+    /* istanbul ignore if */
+    if (subHeader) {
+      subHeader.classList.add("d-none");
```

Comment:
Does this need some kind of cleanup?

Reply:
this is dealing with some django template stuff that we're hiding. @jaredlockhart thoughts?

Reply:
I just meant like, if you are adding a class on mount, do you need a cleanup function for unmount?

Reply:
(maybe not just a thought)

Reply:
Oh oh I wasn't reading closely enough, you're hiding the subheader. Oh we don't need to do that in here, we can do the template, but we can also do that after this lands.  I wouldn't worry too much about it here, leave this how it is, we can land it as a followup.